datasets/gov/pd/eiis.csv

```diff
@@ -0,0 +1,117 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,,,,,,,,,,,,,,
+,datasets/gov/pd/eiis,,,,,,,,,,,,,
+,,json2013,,,,json,,https://data.gov.lt/dataset/509/download/10100/EI_2013%2012%2031.json,,,,,,
+,,json2014,,,,json,,https://data.gov.lt/dataset/509/download/10093/EI_2014%2012%2031.json,,,,,,
+,,json2015,,,,json,,https://data.gov.lt/dataset/509/download/10094/EI_2015%2012%2031.json,,,,,,
+,,json2016,,,,json,,https://data.gov.lt/dataset/509/download/10095/EI_2016%2012%2031.json,,,,,,
+,,json2017,,,,json,,https://data.gov.lt/dataset/509/download/10096/EI_2017%2012%2031.json,,,,,,
+,,json2018,,,,json,,https://data.gov.lt/dataset/509/download/10097/EI_2018%2012%2031.json,,,,,,
+,,json2019,,,,json,,https://data.gov.lt/dataset/509/download/10098/EI_2019%2012%2031.json,,,,,,
+,,json2020,,,,json,,https://data.gov.lt/dataset/509/download/10099/EI_2020%2012%2031.json,,,,,,
+,,,,EismoIvykis,,,registrokodas,,,,,,,
+,,,,,registrokodas,string,,registrokodas,,3,open,,Registro kodas,
+,,,,,data_laikas,datetime,,dataLaikas,,3,open,,Įvykio data ir laikas,
+,,,,,registravimo_data,datetime,,registravimoData,,3,open,,Registravimo data,
+,,,,,paskutinio_redagavimo_laikas,datetime,,paskutinioRedagavimoLaikas,,3,open,,Paskutinio redagavimo data,
+,,,,,iskaitinis,integer,,iskaitinis,,3,open,,Iskaitinis,
+,,,,,rusis,string,,rusis,,3,open,,Rušis,
+,,,,,schema1,string,,schema1,,3,open,,Schema 1,
+,,,,,schema2,string,,schema2,,3,open,,Schema 2,
+,,,,,dalyviu_skaicius,integer,,dalyviuSkaicius,,3,open,,Dalyvių skaičius,
+,,,,,zuvusiu_skaicius,integer,,zuvusiuSkaicius,,3,open,,Žuvusių skaičius,
+,,,,,zuvo_vaiku,integer,,ZuvVaiku,,3,open,,Žuvusių vaikų skaičius,
+,,,,,suzeistu_skaicius,integer,,suzeistuSkaicius,,3,open,,Sužeistų skaičius,
+,,,,,suzeista_vaiku,integer,,suzeistaVaiku,,3,open,,Sužeistų vaikų skaičius,
+,,,,,tp_skaicius,integer,,tpSkaicius,,3,open,,Transporto priemonių skaičius,
+,,,,,policijos_tp_skaicius,integer,,policijosTpSkaicius,,3,open,,Policijos transporto priemonių skaičius,
+,,,,,apgadintu_tp_skaicius,integer,,apgadintuTpSkaicius,,3,open,,Apgadintų transporto priemonių skaičius,
+,,,,,apgadintu_policijos_tp_skaic,integer,,apgadintuPolicijosTpSkaicius,,3,open,,Apgadintų policijos transporto priemonių skaičius,
+,,,,,policijos_istaiga_l1,string,,policijosIstaigaL1,,3,open,,Policijos įstaiga 1,
+,,,,,policijos_istaiga_l2,string,,policijosIstaigaL2,,3,open,,Policjos įstaiga 2,
+,,,,,iforminusi_istaiga_l1,string,,iforminusiIstaigaL1,,3,open,,Įforminusi įstaiga 1,
+,,,,,iforminusi_istaiga_l2,string,,iforminusiIstaigaL2,,3,open,,Įformunusi įstaiga 2,

```

Comment:
Ar čia neturėtu būti ryšys su juridinių asmenų registru?

https://data.gov.lt/dataset/is-juridiniu-asmenu-registro-isregistruoti-juridiniai-asmenys